Modules/Common/include/Common/BigScreen.h

```diff
@@ -48,16 +44,14 @@ class BigScreen : public quality_control::postprocessing::PostProcessingInterfac
   void finalize(quality_control::postprocessing::Trigger, framework::ServiceRegistryRef) override;
 
  private:
+  /// \brief maximum allowed age of quality objects, in seconds (default: 10 minutes)
+  int mMaxObjectTimeShift{ 600 };
+  /// \brief read quality objects from all runs
+  int mIgnoreActivity{ 0 };
```

Comment:
```suggestion
  bool mIgnoreActivity{ false };
```
Why not make it a bool for clarity?

Reply:
Good point, done.